.gitignore

```diff
@@ -48,7 +48,14 @@ TEST-*
 .sass-cache/
 
 # Specific generated JS files
+kive/librarian/librarian.tests.js
+kive/pipeline/pipeline.tests.js
 kive/pipeline/tests.js
 kive/pipeline/tests/*.js
+kive/portal/portal.tests.js
```

Comment:
It's a pain to maintain this list of generated .js files that gets ignored. I wonder if we should move the last few .js files to a lib folder somewhere, and then ignore `kive/**/*.js`.